rsts/deployment/multicluster.rst

```diff
@@ -9,16 +9,18 @@ Scaling Beyond Kubernetes
 -------------------------
 
 .. tip::
-  As described in the `Architecture Overview <https://docs.flyte.org/en/latest/concepts/architecture.html>`_, the Flyte ``Control Plane`` sends workflows off to the ``Data Plane`` for execution. The data-plane fulfills these workflows by launching pods in Kubernetes.
+  As described in the `Architecture Overview <https://docs.flyte.org/en/latest/concepts/architecture.html>`_, the Flyte ``Control Plane`` sends workflows off to the ``Data Plane`` for execution. The data plane fulfills these workflows by launching pods in Kubernetes.
 
-Often, the total compute needs could exceed the limits of a single Kubernetes cluster. 
-To address this, you can deploy the data-plane to several isolated Kubernetes clusters.
-The control-plane (FlyteAdmin) can be configured to load-balance workflows across these isolated data-planes, which protects you from failure in a single Kubernetes cluster, increasing scalability.
+Often, the total compute needs could exceed the limits of a single Kubernetes cluster.
+To address this, you can deploy the data plane to several isolated Kubernetes clusters.
+The control-plane (FlyteAdmin) can be configured to load-balance workflows across these isolated data planes, which protects you from failure in a single Kubernetes cluster, increasing scalability.
```

Comment:
Is this control plane or control-plane?

Reply:
```suggestion
The control plane (FlyteAdmin) can be configured to load-balance workflows across these isolated data planes, protecting you from failure in a single Kubernetes cluster increasing scalability.
```

Reply:
I think it has to be "control plane". Let's use this for now.

Also, let's make the following sentence clear:

... protecting you from failure in a single Kubernetes cluster, thus increasing scalability.